zulipterminal/core.py

```diff
@@ -213,6 +215,34 @@ def show_about(self) -> None:
                       footlink_enabled=self.footlinks_enabled)
         )
 
+    def show_user_info(self, user_id:int) -> None:
+        response = self.client.get_user_by_id(user_id)
+        # TODO: add custom fields later as an enhancement
+        display_data = {}
+        res_data = response['user']
+        display_data['Name'] = res_data['full_name']
+        display_data['Email'] = res_data['email']
+        display_data['Date joined'] = res_data['date_joined'][:10]
+        display_data['Timezone'] = res_data['timezone']
+        try:
+            display_data['Local time'] = datetime.now(pytz.timezone(str(res_data['timezone']))).strftime("%H:%M")
+        except:
+            display_data['Local time'] = "Unknown timezone"
+        presence = self.client.get_user_presence(str(res_data['email']))
```

Comment:
Yeah, I think so (not completely sure though).
By the GET request `client.get_user_by_id(user_id)`, I got the following dict:
```
{
 'result': 'success',
 'msg': '',
'user': 
  {
    'email': 'user16790@chat.zulip.org', 'user_id': 16790, 'avatar_version': 2,
    'is_admin': False, 'is_owner': False, 'is_guest': False, 'is_bot': False,
    'full_name': 'Matt Hall', 'timezone': 'America/Los_Angeles',
    'is_active': True, 'date_joined': '2020-07-24T21:53:55.770840+00:00',
     'avatar_url': '/user_avatars/2/848f00cf86435f7a12c7f6dafb476e47a7ee5665.png?x=x&version=2'
  }
}
```
Seems like we *have* to fetch the timezone of a user in any case. Please drop any suggestions if you have :)

Reply:
Right, well the user data should be available in the model from what I can tell. It would benefit from refactoring, but the initial data is in `Model.initial_data['realm_users']`.

Reply:
Thanks @neiljp for reminding me of the users realm, I've updated the PR accordingly :)